samples/subsys/usb/console/tinytile.overlay

```diff
@@ -0,0 +1,5 @@
+/ {
+        chosen {
+                zephyr,console = &usb_cdc;
+        };
+};
```

Comment:
How to include these overlays to build?

Reply:
this is included by the build system directly.